### ISSUE TITLE
Another compile issue after latest git pull

### ISSUE THREAD

**Issue by trsbox** [dr4fters/dr4ft#347]
> draft@0.2.1 postinstall /home/scorch/dr4ft
> node scripts/postinstall && webpack --config webpack.prod.js

fs.js:885
  return binding.mkdir(pathModule._makeLong(path),user:trsbox dr4ft
                 ^

TypeError: path must be a string or Buffer
    at Object.fs.mkdirSync (fs.js:885:18)
    at Object.<anonymous> (/home/scorch/dr4ft/src/logger.js:7:6)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/scorch/dr4ft/src/make/allsets.js:5:16)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! draft@0.2.1 postinstall: `node scripts/postinstall && webpack --config webpack.prod.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the draft@0.2.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-02-22T20_28_45_214Z-debug.log

**Comment by HerveH44** [comment]
Hi,

Thanks for using our code. Yeah unfortunately one the commit requires that you modify a bit your config files before starting the server. The best you can do is store your config files elsewhere (move them out of the install directory) and restart "npm install". That will recreate new config.server.js and config.client.js. You can then remodify these files with your configuration (name, code etc). 

In fact, I added a logger (that will write the state of the server in the directory data/log by default). You can see its configuration in the file (config/config.server.js.default). That's what is missing in your case in the config.server.js: https://github.com/dr4fters/dr4ft/blob/2bc20cddf02eb860c219c47ae03dbf07dd10589a/config/config.server.js.default#L37

**Comment by HerveH44** [comment]
@trsbox how is it going? Have you been able to make it work?

Just for your information, we're working on a new PR that will force server to have NodeJS version >= 11.x
check your version to be sure to be able to use dr4ft!

**Comment by HerveH44** [comment]
Hi @trsbox , I'm closing this issue now. If you have any new problem, don't hesitate to open a new issue!
